arangod/GeneralServer/CommTask.cpp

```diff
@@ -447,21 +447,18 @@ void CommTask::executeRequest(std::unique_ptr<GeneralRequest> request,
   TRI_ASSERT(request != nullptr);
   TRI_ASSERT(response != nullptr);
 
+  if (request == nullptr || response == nullptr) {
```

Comment:
We have asserts

Reply:
Yes, but the asserts will not fire in production code. That's why I also added the exception.
The previous code contained some `if (request) ... else if (response) ...`, also these things were present only after dereferencing the pointers :+1: